### DISCUSSION TITLE
Defining field names and aliases, domains when exporting to fGDB

### DISCUSSION THREAD

**Discussion by Nova-Scotia** [R-ArcGIS/arcgisbinding#87] [General]
Is there is a method to specify field aliases, and domains when exporting from R to fGDB using `arc.write`?

**Comment by dpavlushko** [comment]
No. `arc.write` works with common R types such as data.frame, vector or raster. They do not carry such information. Supporting these properties as an additional parameter is possible, but it makes the API extremely confusing.

**Reply by Nova-Scotia** [reply]
Thanks - always good to know! 
:)
